pilot/pkg/security/authz/builder/testdata/action-deny-HTTP-for-TCP-filter-out.yaml

```diff
@@ -198,21 +198,11 @@ typedConfig:
                       safeRegex:
                         googleRe2: {}
                         regex: .*/ns/ns/.*
-                - authenticated:
```

Comment:
Why are these removed in the output?

Reply:
I'm not sure. It's removed after I run `make gen`  I'll recover it in next commit. Thanks!